### DISCUSSION TITLE
metaSpades can't input two or more files

### DISCUSSION THREAD

**Discussion by siduanmiao** [ablab/spades#1044] [Q&A]
### Description of bug

I want to use spades.py --meta to do co-assembly but I find that I can't input tow or more files. In the past issues you mentioned that we should make this samples as a single library, so I want to know: Is it different if I cat them as a single fasta.gz or using "--pe" to input many fastq files. 

I don't really understand what is co-assembly,  just cat the files into a file and assembly or using other method such as using the similar features of other samples to improve the length of this sample. 

### spades.log

NO LOG

### params.txt

NO params.txt

### SPAdes version

SPAdes genome assembler v3.14.1

### Operating System

CentOS Linux 7

### Python Version

python 3.9

### Method of SPAdes installation

conda

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
Hello

Well, if you do not really know what co-assembly is, why do you want to do it in the first place? :) There is no difference from specifying multiple files for a single library compared to merging them by hands

**Reply by siduanmiao** [reply]
Thanks!!My professor told me that the spades is the most popular tool for metagenome assembly, and asked me to use metaspades to assembly our group's 16 samples last week, but I am the new of this field. Anyway, thank you a lot, and also your group's excellent job!
